bin/push-to-dockerhub.sh

```diff
@@ -28,3 +28,4 @@ docker push  $CCP_IMAGE_PREFIX/crunchy-scheduler:$CCP_IMAGE_TAG
 docker push $CCP_IMAGE_PREFIX/crunchy-upgrade:$CCP_IMAGE_TAG
 docker push $CCP_IMAGE_PREFIX/crunchy-pgbench:$CCP_IMAGE_TAG
 docker push $CCP_IMAGE_PREFIX/crunchy-pgbasebackup-restore:$CCP_IMAGE_TAG
+docker push  $CCP_IMAGE_PREFIX/crunchy-postgres-ha:$CCP_IMAGE_TAG
```

Comment:
@andrewlecuyer crunchy-postgres-gis-ha too?

Reply:
`crunchy-postgres-gis-ha` is now included